docker/catalog-full/docker-compose.yml

```diff
@@ -29,23 +29,20 @@ services:
       - "19120:19120"
     depends_on:
       jaeger:
-        # This should ideally be 'service_healthy', but that doesn't work on macOS
-        condition: service_started
+        condition: service_healthy
       minio:
-        # This should ideally be 'service_healthy', but that doesn't work on macOS
-        condition: service_started
-      #minio-setup:
-      #  condition: service_started
+        condition: service_healthy
+      minio-setup:
```

Comment:
Please remove this dependency, because if minio-setup completes too fast, Nessie won't start